### DISCUSSION TITLE
Why `getInjectedConnector` is not exported to the user?

### DISCUSSION THREAD

**Discussion by himanshu-leap** [rainbow-me/rainbowkit#1835] [Q&A]
Was there any reason to not export `getInjectedConnector` to the user?

**Comment by 0xmkh** [comment]
@himanshu-leap We don't expose `getInjectedConnector` since it's only used for when creating RainbowKit wallets internally. In what case do you need `getInjectedConnector` for ? We could expose that if you want 🙂

**Reply by jiangmaniu** [reply]
@magiziz Can custom wallet connectors only be created as stated in the documentation?
```ts
createConnector: getWalletConnectConnector({ projectId })
```

 Can I create a connector like the built-in wallet?
```ts
createConnector: shouldUseWalletConnect
      ? getWalletConnectConnector({
          projectId,
          walletConnectParameters,
        })
      : getInjectedConnector({ namespace: 'okxwallet' }),
```

Since there is no built-in Binance wallet connection, I want to customize a Binance wallet connection myself.

**Reply by himanshu-leap** [reply]
@magiziz I am working on Sei EVM, and want dApps to add support for the Compass wallet. I am following EIP-1193 standard, and exposing the provider via `window.compassEvm`. That's why I need this function to be exposed.
`createConnector: getInjectedConnector({ target: window.compassEvm })`

**Reply by himanshu-leap** [reply]
Also, do you accept external PRs in this repo? what if I want to raise a PR to add support for Compass Wallet?
Or, only you can raise a PR? 
Also, If you could share, what's the average time it takes to merge a PR and to make the PR changes live?

**Reply by 0xmkh** [reply]
@himanshu-leap Yeah you can use `getInjectedConnector`, but then you would have to go into our repo and create a wallet. You can of course contribute if you would like, but if you want to i can also add the compassWallet for you ? 

Either way, but if you're choosing to raise a PR on your end you can fork it and raise a PR to our main branch.

**Reply by himanshu-leap** [reply]
Let me raise a PR. I will also expose getInjectedConnector in this PR for someone to use in the future?

**Reply by himanshu-leap** [reply]
@magiziz Please review and merge this PR - https://github.com/rainbow-me/rainbowkit/pull/1837

**Reply by hsnice16** [reply]
I have exposed `getInjectedConnector` in this.

**Reply by 0xmkh** [reply]
@himanshu-leap @hsnice16 Thank you guys 🙏. I'll review it shortly and update our docs as well to include `compassWallet`

**Reply by hsnice16** [reply]
Thanks a lot.

**Reply by hsnice16** [reply]
Please let me know once this goes live. Any ETA on that?

**Reply by 0xmkh** [reply]
Hey let's keep the discussion in the PR [here](https://github.com/rainbow-me/rainbowkit/pull/1837). Left a small comment.

**Reply by hsnice16** [reply]
Okay. I have replied on that comment.
